ctw/fall_of_babylon/map.xml

```diff
@@ -2377,14 +2437,16 @@
                 <rectangle min="-141,-119" max="142,-74"/> <!-- blue main square -->
                 <rectangle min="-100,-15" max="101,16"/> <!-- central gold spawner -->
             </union>
-            <union id="reds-woolrooms">
-                <rectangle id="lime-woolroom" min="-142,146" max="-115,120"/>
-                <rectangle id="cyan-woolroom" min="116,146" max="143,120"/>
-            </union>
-            <union id="blues-woolrooms">
-                <rectangle id="orange-woolroom" min="143,-146" max="116,-119"/>
-                <rectangle id="yellow-woolroom" min="-115,-146" max="-142,-119"/>
-            </union>
+            <negative id="not-woolrooms">
+                <union id="reds-woolrooms">
+                    <rectangle id="lime-woolroom" min="-142,146" max="-115,120"/>
+                    <rectangle id="cyan-woolroom" min="116,146" max="143,120"/>
+                </union>
+                <union id="blues-woolrooms">
+                    <rectangle id="orange-woolroom" min="143,-146" max="116,-119"/>
+                    <rectangle id="yellow-woolroom" min="-115,-146" max="-142,-119"/>
+                </union>
+            </negative>
```

Comment:
```suggestion
            <negative id="not-woolrooms">
                <union id="woolrooms">
                    <union id="reds-woolrooms">
                        <rectangle id="lime-woolroom" min="-142,146" max="-115,120"/>
                        <rectangle id="cyan-woolroom" min="116,146" max="143,120"/>
                    </union>
                    <union id="blues-woolrooms">
                        <rectangle id="orange-woolroom" min="143,-146" max="116,-119"/>
                        <rectangle id="yellow-woolroom" min="-115,-146" max="-142,-119"/>
                    </union>
                </union>
            </negative>
```

Reply:
> 

Thank you for these simplifications.  They have been applied now.